src/components/FilterList.js

```diff
@@ -0,0 +1,54 @@
+const FilterList = ({
+                        isCompletedStatus,
+                        isInProgressStatus,
+                        statusList,
+                        setStatusList
+                    }) => {
+
+    return (
+      <>
+          <div className="form-check">
+              <input className="form-check-input" type="checkbox" value=""
```

Comment:
Let's move all attributes on the next line (in all such cases)
example:

<input
            className="form-check-input" 
            type="checkbox" 
            value=""

Reply:
done